Automation Examples/020.md

```diff
@@ -0,0 +1,161 @@
+# Process
+
+Open https://www.vat19.com/, go to fun & strange stuff and download first 20 products' information (name, price, image) and save it into excel file. Template is available [here](Products%20Template.xlsx).
+
+# Steps to automate
+
+#### 1. Specify a path to your folder, open website and maximise the browser
+
+```G1ANT
+♥basePath = C:\Users\dell\Desktop\Automation Examples
+selenium.open chrome url https://www.vat19.com/
+window ✱Unique✱ style maximize
+```
+
+Use window command and part of the window title as an argument to bring it to the front and maximise.
+
+#### 2. Go to fun & strange stuff, create lists which will contain product names and their prices
+
+After running the above script and seeing the webpage, you can inspect the button element and in the developer tools, right click it and choose "Copy Xpath" that you can insert in the search argument for selenium.click command.
```

Comment:
I don't see any button element.. :( Elaborate on this; it's hard to find out what to do.
![image](https://user-images.githubusercontent.com/7533717/64418971-3be74a80-d09c-11e9-90bd-155a9839b6aa.png)

Reply:
In Automation Anywhere example it wasn't described. Users are told to open "Fun & Strange Stuff" webpage so it's obvious that you need to click this button while being on the main webpage.

Reply:
i'm just saying that it is not a *button*, it's just navigation menu item

Reply:
```suggestion
After running the above script and seeing the webpage, you can inspect the "Fun & Strange Stuff" element and in the developer tools, right click it and choose "Copy Xpath" that you can insert in the search argument for selenium.click command.
```